resources/quality/ultimaker_s5/um_s5_aa0.4_petg_0.3mm.inst.cfg

```
@@ -0,0 +1,21 @@
+[general]
+definition = ultimaker_s5
+name = Extra Fast - Experimental
+version = 4
+
+[metadata]
+is_experimental = True
+material = generic_petg
+quality_type = verydraft
+setting_version = 22
+type = quality
+variant = AA 0.4
+weight = -3
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```